server/apps/immich/src/api-v1/asset/asset.service.ts

```diff
@@ -188,16 +188,12 @@ export class AssetService {
     isVisible: boolean,
     livePhotoAssetEntity?: AssetEntity,
   ): Promise<AssetEntity> {
-    // Check valid time.
-    const createdAt = createAssetDto.createdAt;
-    const modifiedAt = createAssetDto.modifiedAt;
-
-    if (!timeUtils.checkValidTimestamp(createdAt)) {
-      createAssetDto.createdAt = await timeUtils.getTimestampFromExif(originalPath);
+    if (!timeUtils.checkValidTimestamp(createAssetDto.createdAt)) {
+      createAssetDto.createdAt = new Date().toISOString();
     }
 
-    if (!timeUtils.checkValidTimestamp(modifiedAt)) {
-      createAssetDto.modifiedAt = await timeUtils.getTimestampFromExif(originalPath);
+    if (!timeUtils.checkValidTimestamp(createAssetDto.modifiedAt)) {
+      createAssetDto.modifiedAt = new Date().toISOString();
```

Comment:
This value is no longer coming from exif. Do we want to update it when we save `createdAt` in the processor?